server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java

```diff
@@ -126,14 +125,19 @@ public static Club getClub(DatabaseClient dbClient, String clubId) {
           .readRow(
             "Books",
             Key.of(clubId),
-            Arrays.asList("bookId", "name", "description", "ownerId"));
+            Arrays.asList("bookId", "name", "description", "ownerId", "contentWarning"));
     if (row != null) {
-      Book book = getBook(dbClient, row.getString(/*bookIdIndex=*/0));
-      Club.Builder clubBuilder = Club.newBuilder(row.getString(/*nameIndex=*/1),
-                                                 book);
-      // TODO: implement setting the clubId field @JosephBushagour
-      // TODO: implement setting the ownerId field @JosephBushagour
-      clubBuilder.setDescription(row.getString(/*descriptionIndex=*/2));
+      Book book = getBook(dbClient, row.getString(/* bookIdIndex= */ 0));
+      Club.Builder clubBuilder = Club.newBuilder()
+                                     .setCurrentBook(book)
+                                     .setName(row.getString(/* nameIndex= */ 1))
+                                     .setClubId(clubId)
+                                     .setDescription(row.getString(/* descriptionIndex= */ 2))
+                                     .setOwnerId(row.getString(/* ownerIdIndex= */ 3));                       
+      if (!row.isNull(/* index= */ 4) || !row.getString(/* index= */ 4).isEmpty()) {
```

Comment:
nit: contentWarningIndex

Reply:
Good catch! This has been fixed and pushed.